<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulSettings</class>
 <widget class="QDialog" name="SimulSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>296</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>32767</width>
    <height>32767</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Simulation Settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/new/prefix1/images/appicon.png</normaloff>:/new/prefix1/images/appicon.png</iconset>
  </property>
  <layout class="QGridLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Error Options</string>
       </property>
       <layout class="QGridLayout">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkBox1">
            <property name="toolTip">
             <string>Enable/Disable G Code Syntax Checking</string>
            </property>
            <property name="text">
             <string>Syntax Enabled</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox2">
            <property name="text">
             <string>Stop At Intersection</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Tool Polygonization/Resolution</string>
       </property>
       <layout class="QGridLayout">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QSlider" name="slider1">
            <property name="maximum">
             <number>3</number>
            </property>
            <property name="pageStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>2</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="textLabel1">
              <property name="text">
               <string>Low</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="textLabel2">
              <property name="text">
               <string>Medium</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="textLabel3">
              <property name="text">
               <string>High</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="textLabel4">
              <property name="text">
               <string>Very High</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="TextLabel1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="text">
          <string>Rapid Rate</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="SpinBoxRapid">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Enter The Rapid Move Rate</string>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::UpDownArrows</enum>
         </property>
         <property name="suffix">
          <string>mm/min</string>
         </property>
         <property name="minimum">
          <number>1000</number>
         </property>
         <property name="maximum">
          <number>99999</number>
         </property>
         <property name="value">
          <number>10000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="TextLabel2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="text">
          <string>Points/Frame</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="SpinBoxPPF">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Enter The Number Of Points Per Frame</string>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::UpDownArrows</enum>
         </property>
         <property name="suffix">
          <string>Points</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="OKButton">
         <property name="text">
          <string>OK</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>image1</normaloff>image1</iconset>
         </property>
         <property name="shortcut">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="CancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>image2</normaloff>image2</iconset>
         </property>
         <property name="shortcut">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="helpButton">
         <property name="text">
          <string>&amp;Help</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>image3</normaloff>image3</iconset>
         </property>
         <property name="shortcut">
          <string>Alt+H</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>SpinBoxRapid</tabstop>
  <tabstop>SpinBoxPPF</tabstop>
  <tabstop>OKButton</tabstop>
  <tabstop>CancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SimulSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OKButton</sender>
   <signal>clicked()</signal>
   <receiver>SimulSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
